### PR TITLE
New step button design

### PR DESCRIPTION
**Pull request by jorolf**
I haven't changed the `Light`  colour because I was unsure which colours to use (some aren't even in the new design)

### CODE REVIEW

File: osu.Framework/Testing/Drawables/Steps/StepButton.cs [hunk 1]
```diff
@@ -64,9 +66,10 @@ protected StepButton()
                 {
                     Anchor = Anchor.CentreLeft,
                     Origin = Anchor.CentreLeft,
-                    Font = new FontUsage(size: 14),
+                    Font = new FontUsage("Roboto", 14, "Regular"),
```
Comment (peppy) [inline osu.Framework/Testing/Drawables/Steps/StepButton.cs]: `FrameworkFont` now exists (ditto for other usages below)

**Comment by peppy** [comment]
Headers really don't work right now

image

@arflyte poke

**Comment by arflyte** [comment]
New header

Illustrator_190702-131322

**Comment by jorolf** [comment]
No yellow buttons anymore?

**Comment by peppy** [comment]
Yep, we went away from yellow.

**Pull request merged**